src/openms/source/FORMAT/MSPFile.cpp

```diff
@@ -195,6 +179,21 @@ namespace OpenMS
             // e.g. Mods=2/7,K,Carbamyl/22,K,Carbamyl
             vector<String> mod_split;
             mods.split('/', mod_split);
+            if (mod_split.size() <= 1) // e.g. Mods=2(0,A,Acetyl)(11,M,Oxidation)
```

Comment:
<=1 or == 1?

Reply:
should be smaller. but not sure in which case it would be zero. but one should try the second case anyway